doc/articles/migrating-from-previous-releases.md

```diff
@@ -41,6 +41,26 @@ A few considerations to keep in mind:
 
 For an up-to-date list of **known issues** when using **.NET 10** with Uno Platform, please refer to our [Health Status page](https://aka.platform.uno/health-status).
 
+## API Changes for Android
+
+Starting with Uno Platform **6.3**, the following methods now throw `NotSupportedException` under **.NET 10** or later:
+
+- `Microsoft.UI.Xaml.ApplicationActivity.GetTypeAssemblyFullName(string)` (Android + Native renderer)
+- `Microsoft.UI.Xaml.ApplicationActivity.GetTypeAssemblyFullName(string)` (Android + Skia renderer)
+- `Microsoft.UI.Xaml.NativeApplication.GetTypeAssemblyFullName(string)`
+
+This change was introduced in [PR #21199](https://github.com/unoplatform/uno/pull/21199) ([commit 4d84ee3](https://github.com/unoplatform/uno/commit/4d84ee31adb3e7ecd3fcbdc248b79fee78211d3e)) to improve **.NET 10 NativeAOT** support.
```

Comment:
~~to improve **.NET 10 NativeAOT** support~~.

(i.e. *yes*, leave the rationale "unspecified" except for those who follow the PR link.)